src/redux/actions/moviesAction.ts

```diff
@@ -55,3 +95,120 @@ const getAllMoviesError = (error: Error) => ({
     type: GET_MOVIES_ERROR,
     payload: error
 });
+
+export function createNewMovieAction(info: Movie, setIsOpen: (newState: boolean) => void) {
+    return async(dispatch: AppDispatch) => {
+        dispatch(createMovieStart());
+        
+        try {
+            const response = await fetch(`${ BACKEND_URL }`, {
+                method: "POST",
+                mode: "cors",
```

Comment:
You use this options for all of your requests. You can use it like this:
```js
const COMMON_OPTIONS = {
  mode: "cors",
  cache: "no-cache",
  credentials: "same-origin",
  headers: {
    'Content-Type': 'application/json'
  }
}
```

Reply:
Resolved